patches/server/0005-Paper-config-files.patch

```diff
@@ -719,6 +719,13 @@ index 0000000000000000000000000000000000000000..8633fdfa7e8aeac7414c232006fc298b
 +        public boolean useDimensionTypeForCustomSpawners = false;
 +        public boolean strictAdvancementDimensionCheck = false;
 +    }
++
++    public BlockUpdates blockUpdates;
++
++    public class BlockUpdates extends ConfigurationPart {
++        public boolean disableNoteblockUpdates = false;
++        public boolean disableTripwireUpdates = false;
++    }
```

Comment:
Since they are unsupported settings, This section should be in the unsupported section.

Also, should this go in the WorldConfiguration? Is there any benefit you see as a user of custom blocks to having it per world?

Reply:
Was told it shouldnt be in unsupported by kenny, hence the change. or it could "go outside of unsupported". See #9367 
also I thought about making it per world, but I don't see any real benefit as the plugins that would implement this most likely doesnt distinguish between worlds for the feature.

The resourcepack would have to be changed between each world then. Since the blockstate files dictate the "model" shown, a world where this would be disabled would have very funky looking tripwires/noteblocks
Might be benefits to making it per-world though that im not seeing

Reply:
Ignore the bit about unsupported. They are fine in their own section. But my question about per world still stands.